news/2023/2023-12-12-how-advances-in-technology-affected-the-osu-community.md

```diff
@@ -0,0 +1,101 @@
+
+It's time to dive in and see how technology can impact the community, what the results of widespread adoption are, or could have been.
+
+## Graphics tablets (2007)
+
+osu! originally started out as an *[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan)* clone for PC, which was a game that was exclusive to the Nintendo DS, a console that came, and could be used with the included stylus pen. When [peppy](https://osu.ppy.sh/users/2) released osu!, it was only natural for players who already had a graphics tablet to try out osu! with it, and since the get-go it was considered a commodity input device that would more closely replicate the original Nintendo DS experience.
+
+Playing with a tablet is a completely different ball game compared to the mouse, but rather than being a direct upgrade by eliminating mouse drift with its absolute positioning, it brings up new challenges and questions regarding its ideal usage. Ideal pen grip, tablet area, dragging or hovering play styles have all been detrimental to increasing the learning curve, which is why some players choose to remain on the much more familiar and intuitive mouse.
+
+Over the years many top players and regular players started switching over from mouse to tablet in order to avoid issues associated with mice, however there have always been top players with both input devices, and they are both considered viable for top-level play, each with its own upsides and downsides.
+
+A great video by community member [Willy](https://osu.ppy.sh/users/3521482) outlines what's written here and more regarding the usage of tablets as input devices:
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/V3K3C94fs1o" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Gaming mechanical keyboards (2010)
+
+Mechanical keyboards have been a thing for decades, however after Razer put [gaming and mechanical switches into a compelling package](https://press.razer.com/product-news/the-worlds-first-mechanical-keyboard-designed-for-gaming-launched-at-gamescom-2010/), that launched the chain reaction that resulted in the mechanical keyboard market we know of today.
+
+By 2010 osu! players were already shifting away from clicking circles with their mouse, to clicking with their Z and X keys, so it would make sense that as these new mechanical keyboards were getting easier and cheaper to adopt into their setups, players would start using them for osu! as well. And while there's nothing stopping anyone from playing osu! with the membrane keyboards, the community at large is suggesting to newer players to purchase a mechanical keyboard, as it provides better feedback when a key is pressed.
```

Comment:
```suggestion
By 2010 osu! players were already shifting away from clicking circles with their mouse to mashing their Zs and Xs, so it would make sense that as these new mechanical keyboards were getting easier and cheaper to adopt into their setups, players would start using them for osu! as well. And while there's nothing stopping anyone from playing osu! with the membrane keyboards, the community at large is guiding the newer players to purchase a mechanical keyboard, as it provides better feedback, which is essential to the rhythm-based gameplay.
```

Reply:
https://github.com/ppy/osu-wiki/pull/10659/files#r1428978830 combined the two pieces of feedback